CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * [ENHANCEMENT] Distributor: Added distributors ring status section in the admin page. #4151
 * [BUGFIX] Purger: fix `Invalid null value in condition for column range` caused by `nil` value in range for WriteBatch query. #4128
 * [BUGFIX] Ingester: fixed infrequent panic caused by a race condition between TSDB mmap-ed head chunks truncation and queries. #4176
+* [ENHANCEMENT] Added `tenant_ids` tag to tracing spans
```

Comment:
Also please move it above, along with the group of `[ENHANCEMENT]` entries.

```suggestion
* [ENHANCEMENT] Added `tenant_ids` tag to tracing spans. #4186
```

Reply:
done